Device/Device.cs

```diff
@@ -13,7 +13,7 @@
 using MQTTnet.Formatter;
 using MQTTnet.Protocol;
 
-namespace MessageSample
+namespace Client

```

Comment:
Client not Device?

Reply:
I guess Device is already used in the class name

Reply:
Pooh, good point!

Reply:
Yes, could not use the same name for both namespace and type